core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.cpp

```diff
@@ -98,6 +98,11 @@ void OpenMPTargetExec::resize_scratch(int64_t team_size, int64_t shmem_size_L0,
                                       int64_t shmem_size_L1,
                                       int64_t league_size) {
   Kokkos::Experimental::OpenMPTargetSpace space;
+  // FIXME_OPENMPTATGET - Level-0 scratch when using clang/17 and higher comes
```

Comment:
It is not really a FIXME though is it?
Maybe add that the L0 request is performed via `ompx_dyn_cgroup_mem(shmem_size_L0)` in the omp pragma